extensions/amp-accordion/validator-amp-accordion.protoascii

```diff
@@ -13,7 +13,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the license.
 #
-tags: {  # amp-accordion
+
+tags: {  # amp-accordion for websites
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-accordion"
+    version: "1.0"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # amp-accordion for ads
```

Comment:
Updated to match the format of https://github.com/ampproject/amphtml/pull/33478/

This allows removal of the following lines for `1.0`
```
requires_usage: EXEMPTED
deprecated_allow_duplicates: true
```
Enforce -
`1.0` components should not have duplicated script tags and should have a component on the page when script tag is used.  This is not done in `0.1` for backward compatibility reasons.

Reply:
Update: we're waiting on http://cl/366320502 to land in GitHub; adds support for multiple extension versions.

Reply:
Thanks for the update!